### DISCUSSION TITLE
BBC BASIC recompiled to use FabGL graphics primitives

### DISCUSSION THREAD

**Discussion by fdivitto** [fdivitto/FabGL#154] [Show and tell]
https://automatico.freevar.com/index.php/cp-m-80-2/bbc-basic/

**Comment by OzHawk** [comment]
That's really funny.  I only just posted this to Russell's BBC BASIC repository yesterday.
[https://github.com/rtrussell/BBCSDL/issues/12](url)
The version of BBC BASIC in Russell's repository may be a better one to use for the ESP32 as it is already 32bit.
I'm not sure that I have the skills at the moment to complete this activity, but I will support someone who has in any way that I can.
